lib/mobility/backends/active_record/key_value.rb

```diff
@@ -155,6 +155,9 @@ def visit_default(_)
       setup do |attributes, options|
         association_name   = options[:association_name]
         translations_class = options[:class_name]
+        key_column         = options[:key_column]
```

Comment:
For some reason the auto-generated `key_column`, etc methods are not yet available here.

Reply:
I am not experiencing this when using you branch to experiment with the solution.  When I put a debug statement on line 161, I will get this debugger during project boot, as expected and the `association_name`, `translations_class`, `key_column`, `value_column` and `translatable` variables are all filled with the values that I provided in the `mobility.rb` initializer. This looks to be fine!

Reply:
In my latest tests I still needed these lines, so leaving them for now.